### PR TITLE
Add 'libusb' package

### PR DESCRIPTION
**Pull request by sjdrc**
* I've followed [this guide](https://docs.hunter.sh/en/latest/creating-new/create/cmake.html)
  step by step carefully. **[Yes]**

* I've tested this package remotely and have excluded all broken builds.
  Here is the links to the Travis/AppVeyor with status "All passed":

  * https://ci.appveyor.com/project/sjdrc/hunter/builds/24014902
  * https://travis-ci.com/black-ai/hunter/builds/109137184

* I have submitted CI configs to https://github.com/ingenue/hunter targeting `pkg.template` branch,
  see this merged pull request https://github.com/ingenue/hunter/pull/358

* My change will resolve this "New package" request: https://github.com/ruslo/hunter/issues/1732
* My change will also resolve this related issue: 1828

---

### CODE REVIEW

File: docs/packages/pkg/libusb.rst [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  :language: cmake
+  :start-after: # DOCUMENTATION_START {
+  :end-before: # DOCUMENTATION_END }
+
+Pitfalls
+--------
+
+- Requirements for Ubuntu:
+
+.. code::
```
Comment (ruslo) [inline docs/packages/pkg/libusb.rst]: Use:
```
.. code:: none
```

- https://travis-ci.org/ruslo/hunter/builds/523035503#L2032

File: docs/packages/pkg/libusb.rst [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+.. index::
+  single: unsorted ; libusb
+
+.. _pkg.libusb:
+
+libusb
+======
+
+-  `Official <https://libusb.info>`__
+-  `Hunterized <https://github.com/libusb/libusb>`__
```
Comment (ruslo) [inline docs/packages/pkg/libusb.rst]: It's not "Hunterized", it's "Official GitHub".

**Comment by sjdrc** [comment]
About to push a fix now, Can I just add a commit, or shall I squash it into the first one?

File: docs/packages/pkg/libusb.rst (hunk 2 above)
Comment (sjdrc) [reply docs/packages/pkg/libusb.rst]: Fixed.

**Comment by ruslo** [comment]
Up to you. Since it's an adding of the package you can squash everything into "Add 'libusb' package".

File: docs/packages/pkg/libusb.rst (hunk 1 above)
Comment (sjdrc) [reply docs/packages/pkg/libusb.rst]: The failure to build was actually due to needing a blank line after `.. code::` and the `apt-get` line

Comment (ruslo) [reply docs/packages/pkg/libusb.rst]: Ok, I think I confused it with `code-block`.

**Comment by ruslo** [comment]
Testing:
- https://ci.appveyor.com/project/ingenue/hunter/builds/24016115
- https://travis-ci.org/ingenue/hunter/builds/523051494?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Uploading:
- https://ci.appveyor.com/project/ingenue/hunter/builds/24016404
- https://travis-ci.org/ingenue/hunter/builds/523055889?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Released: https://github.com/ruslo/hunter/releases/tag/v0.23.156

**Pull request merged**